code/game/objects/items/wayfinding.dm

```diff
@@ -86,33 +86,36 @@
 	if(istype(I, /obj/item/pinpointer/wayfinding))
 		var/obj/item/pinpointer/wayfinding/WP = I
 		to_chat(user, "<span class='notice'>You put \the [WP] in the return slot.</span>")
-		var/rfnd_amt
-		if((!WP.roundstart || WP.owner != user.real_name) && synth_acc.has_money(TRUE)) //can't recycle own pinpointer for money if not bought; given by a neutral quirk
-			if(synth_acc.has_money(refund_amt))
-				rfnd_amt = refund_amt
-			else
-				rfnd_amt = synth_acc.account_balance
-			synth_acc._adjust_money(-rfnd_amt)
-			var/obj/item/holochip/HC = new /obj/item/holochip(user.loc)
-			HC.credits = rfnd_amt
+		var/money = TRUE
+		if(synth_acc.has_money(refund_amt) && !WP.roundstart && WP.owner != user.real_name) //exploiters bring a friend
+			synth_acc._adjust_money(-refund_amt)
+			var/obj/item/holochip/HC = new /obj/item/holochip(loc)
+			HC.credits = refund_amt
 			HC.name = "[HC.credits] credit holochip"
 			if(istype(user, /mob/living/carbon/human))
 				var/mob/living/carbon/human/H = user
 				H.put_in_hands(HC)
 		else
-			var/crap = pick(subtypesof(/obj/effect/spawner/bundle/costume)) //harmless garbage some people may appreciate
-			new crap(user.loc)
+			money = FALSE
+			var/costume = pick(subtypesof(/obj/effect/spawner/bundle/costume)) //I don't think anyone wants this
+			new costume(user.loc)
 		qdel(WP)
-		set_expression("happy", 2 SECONDS)
-		say("<span class='robot'>Thank you for recycling, [user.first_name()]! Here is [rfnd_amt ? "[rfnd_amt] credits." : "a freshly synthesized costume!"]</span>")
+		set_expression("veryhappy", 2 SECONDS)
+		var/refund = "some credits."
+		if(!money)
+			refund = "a freshly synthesized costume!"
+			if(prob(1))
+				refund = "a pulse rifle! Just kidding it's a costume."
+		say("<span class='robot'>Thank you for feeding me, [user.first_name()]! Here is [refund]</span>") //it loves the costumes
 
 /obj/machinery/pinpointer_dispenser/proc/set_expression(type, duration)
 	cut_overlays()
 	deltimer(expression_timer)
 	add_overlay(type)
 	if(duration)
-		expression_timer = addtimer(CALLBACK(src, .proc/set_expression, "neutral"), duration, TIMER_STOPPABLE)
+		expression_timer = addtimer(CALLBACK(src, .proc/set_expression, "happy"), duration, TIMER_STOPPABLE)
 
+///A glib facsimile of actual pointing
 /obj/machinery/pinpointer_dispenser/proc/pointat(atom)
 	visible_message("<span class='name'>[src]</span> points at [atom].")
 	new /obj/effect/temp_visual/point(atom,invisibility)
```

Comment:
How difficult is it to push `pointat` down to the atom level?

Reply:
That would be good. Pointing no longer behaves like this. It would be good if everything could use the way mobs do it now.